cmd/authelia-scripts/docker.go

```diff
@@ -22,3 +22,12 @@ func (d *Docker) Login(username, password string) error {
 func (d *Docker) Push(tag string) error {
 	return CommandWithStdout("docker", "push", tag).Run()
 }
+
+// Manifest push a docker manifest to dockerhub.
+func (d *Docker) Manifest(tag string, amd64tag string, arm32v7tag string, arm64v8tag string) error {
+	return CommandWithStdout("docker", "manifest", "push", "--purge", tag).Run()
```

Comment:
doing a return at every line is a bug. Only the first command will be executed. The rest is unreachable code.